packages/http-client-csharp/package.json

```
@@ -51,16 +51,16 @@
     "@typespec/versioning": ">=0.50.0 <1.0.0"
   },
   "devDependencies": {
-    "@azure-tools/typespec-azure-core": "0.40.0",
-    "@azure-tools/typespec-client-generator-core": "0.40.0",
+    "@azure-tools/typespec-azure-core": "0.41.0",
+    "@azure-tools/typespec-client-generator-core": "0.42.0-dev.10",
```

```suggestion
    "@azure-tools/typespec-client-generator-core": "0.41.2",
```

I think that got updated to `0.41.2`

---

bump to `0.41.2`, thanks